python_on_whales/components/buildx/cli_wrapper.py

```diff
@@ -441,6 +441,7 @@ def create(
         context_or_endpoint: Optional[str] = None,
         buildkitd_flags: Optional[str] = None,
         config: Optional[ValidPath] = None,
+        platforms: Optional[List[str]] = None,
```

Comment:
What would be nice here is to give the users the list of allowed platforms through typing information. We can do this with `typing.Literal`.

If you can, please find the list of available platforms and put them all in the literal:

`  platforms: Optional[List[Literal["linux/amd64", "linux/arm64"]] = None,` (put all allowed literals here)

Reply:
I borrowed it from [here](https://github.com/gabrieldemarmiesse/python-on-whales/blob/master/python_on_whales/components/buildx/cli_wrapper.py#L224). Is a list of platforms something you would like to maintain?

Reply:
I don't mind maintaining it, it's very low-effort

Reply:
Do you know where to find a definitive list of supported `os/arch/variant`? Docker docs don't mention it https://docs.docker.com/engine/reference/commandline/buildx_build/#platform

Reply:
In general couldn't this change over time, such that old versions of PoW could end up being out of date? I think I'd vote for sticking with strings as with many other fields where we just pass through to the underlying container client.

Reply:
I believe that it should be better to provide hints to users to get a better UX in the IDE (literals provide great autocompletion in VScode for example). But this can be done in another PR, independently. If it's non-trivial, then let's not block this PR because of it.